nix/README.md

```diff
@@ -0,0 +1,149 @@
+# Prism Launcher Nix packaging
+
+## Installing a stable release (nixpkgs)
+
+Prism Launcher is packaged in [nixpkgs](https://github.com/NixOS/nixpkgs/) since 22.11.
+
+See [Package variants](#package-variants) for a list of available packages.
+
+## Installing a development release (flake)
+
+### Using the overlay
+
+After adding `github:PrismLauncher/PrismLauncher` to your flake inputs, you can add the `default` overlay to your nixpkgs instance.
+
+Example:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    prismlauncher = {
+      url = "github:PrismLauncher/PrismLauncher";
+      # Optional: Override the nixpkgs input of prismlauncher to use the same revision as the rest of your flake
+      # inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = {nixpkgs, prismlauncher}: {
+    nixosConfigurations.foo = nixpkgs.lib.nixosSystem {
+      system = "x86_64-linux";
+
+      modules = [
+        ({pkgs, ...}: {
+          nixpkgs.overlays = [prismlauncher.overlays.default];
+
+          environment.systemPackages = [pkgs.prismlauncher];
+        })
+      ];
+    };
+  }
+}
+```
+
+### Installing the package directly
+
+Alternatively, if you don't want to use an overlay, you can install Prism Launcher directly by installing the `prismlauncher` package.
+
+Example:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    prismlauncher = {
+      url = "github:PrismLauncher/PrismLauncher";
+      # Optional: Override the nixpkgs input of prismlauncher to use the same revision as the rest of your flake
+      # inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = {nixpkgs, prismlauncher}: {
+    nixosConfigurations.foo = nixpkgs.lib.nixosSystem {
+      system = "x86_64-linux";
+
+      modules = [
+        ({pkgs, ...}: {
+          environment.systemPackages = [prismlauncher.packages.${pkgs.system}.prismlauncher];
```

Comment:
this is perfectly reproducible assuming that nixpkgs hasn't been overridden. might be worth a mention